addons/website/static/tests/builder/translation.test.js

```diff
@@ -167,7 +167,7 @@ test("translate attribute history", async () => {
     await contains(".modal .modal-body input").edit("titre");
     await contains(".modal .btn:contains(Ok)").click();
     const getImg = ({ titleName, translated }) =>
-        `<img src="/web/image/website.s_text_image_default_image" class="img img-fluid o_editable o_translatable_attribute${translated ? " oe_translated" : ""
+        `<img src="/web/image/website.s_text_image_default_image" class="img img-fluid o_editable o_translatable_attribute o_editable_media${translated ? " oe_translated" : ""
```

Comment:
Note that I tested the change manually and having `o_editable_media` here does _not_ in fact make the media editable, which matches the expected behavior of images in translate mode.